src/track/trackmetadatataglib.cpp

```diff
@@ -1026,6 +1026,19 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
     const TagLib::ID3v2::FrameList bpmFrame(tag.frameListMap()["TBPM"]);
     if (!bpmFrame.isEmpty()) {
         parseBpm(pTrackMetadata, toQStringFirstNotEmpty(bpmFrame));
+        long bpmValue = pTrackMetadata->getBpm().getValue();
+        //Some software uses (or used) to write decimated values without comma,
+        //so the number reads as 1352 or 14525 when it is 135.2 or 145.25
```

Comment:
I believe it might have been when I had BPM Studio 4, around the year 2003.
I have been able to locate the files that have these tags (now that I know that it's a tag) and it's only 113 file and they are mostly of the years 2000 to 2003.

Btw, I found one file (not related to the others, since it's from 2007) that has this value: "130.01" , yes, with one dot.